lib/src/main/java/com/tbruyelle/rxpermissions/Trigger.java

```diff
@@ -0,0 +1,9 @@
+package com.tbruyelle.rxpermissions;
+
+final class Trigger {
+    static final Trigger INSTANCE = new Trigger();
```

Comment:
Null values are forbidden now. Observable.just(null) will throw NPE that's why you need some other type